src/components/views/rooms/ReadReceiptMarker.tsx

```diff
@@ -123,42 +121,85 @@ export default class ReadReceiptMarker extends React.PureComponent<IProps, IStat
         }
     }
 
-    private animateMarker(): void {
-        // treat new RRs as though they were off the top of the screen
-        let oldTop = -15;
+    private buildReadReceiptInfo(target: IReadReceiptInfo = {}): IReadReceiptInfo {
+        const element = this.avatar.current;
+        // this is the mx_ReadReceiptsGroup_container
+        const horizontalContainer = element.offsetParent;
+        if (horizontalContainer == null || !(horizontalContainer instanceof HTMLElement)) {
```

Comment:
Should we favour `===` here instead ? or are we actually checking for falsy values?

Reply:
I think the intention would become more clear if I switch back to `!horizontalContainer`, like the old code used.